projects/aca-content/src/lib/components/dl-custom-components/thumbnail-column/thumbnail-column.component.spec.ts

```diff
@@ -0,0 +1,118 @@
+/*!
+ * Copyright © 2005-2024 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { ThumbnailColumnComponent } from './thumbnail-column.component';
+import { TranslationService } from '@alfresco/adf-core';
+
+describe('ThumbnailColumnComponent', () => {
+  let component: ThumbnailColumnComponent;
+  let fixture: ComponentFixture<ThumbnailColumnComponent>;
+  let translationServiceMock: any;
+
+  beforeEach(async () => {
+    translationServiceMock = {
+      instant: jasmine.createSpy('instant').and.returnValue('Locked by')
+    };
+
+    await TestBed.configureTestingModule({
+      imports: [ThumbnailColumnComponent],
+      providers: [{ provide: TranslationService, useValue: translationServiceMock }]
+    }).compileComponents();
+  });
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(ThumbnailColumnComponent);
```

Comment:
Can we have a single beforeEach? Having multiple beforeEach blocks introduces a very small probability of this beforeEach executing before the other one, which would result in an [error](https://v17.angular.io/guide/testing-components-basics#createcomponent), since methods like configureTestingModule() cannot be called after TestBed has been frozen via createComponent().